fhir-search/src/test/java/com/ibm/fhir/search/uri/test/UriTest.java

```diff
@@ -32,7 +32,7 @@
 public class UriTest {
 
     @Test
-    public void testUriBadSecurity() throws URISyntaxException {
+    public void testUriTrimmedUrl() throws URISyntaxException {
```

Comment:
hate to be a nit... let's update copyright year.

Reply:
its already 2021